<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dx_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dy_input">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dz_input">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>150</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="move_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>move</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xy_ang_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xz_ang_input">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>310</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yz_ang_input">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>230</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kz_input">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ky_input">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kx_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="infile_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="outfile_input">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="read_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="write_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>70</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>write</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
